### ISSUE TITLE
Regularly updated Linux arm build of the program

### ISSUE THREAD

**Issue by robertstrom** [hluk/CopyQ#2599]
**Is your feature request related to a problem? Please describe.**

The current version of the CopyQ program that I can install on a Raspberry Pi is v6.4

This is the 64bit version of the Raspberry Pi OS.

PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

![20240119_17h17m34s_grim](https://github.com/hluk/CopyQ/assets/3240205/91f0e8e4-5e38-43ad-8b53-5ca988ee5b7c)

**Describe the solution you'd like**

I would like to see current versions of CopyQ available for Linux ARM either in the Raspberry Pi repository or available for download from GitHub.

**Describe alternatives you've considered**

**Additional context**

I use CopyQ on many other Linux systems and I backup and restore the data and configurations regularly. Not sure if I will run into issues when copying backups from newer versions and restoring to an older version of the program. Besides that potential issue, I would just like to see current version builds for ARM64 Linux. The program should preferably be built for the 32bit and the 64bit version of the operating system.

**Comment by hluk** [comment]
I've enabled the builds for ARM on openSUSE Build Service. Here are the builds:

- [armhf/armv7l](https://download.opensuse.org/repositories/home:/lukho:/copyq/Raspbian_12/armhf/)
- [arm64/aarch64](https://download.opensuse.org/repositories/home:/lukho:/copyq/Raspbian_12/arm64/)

Can you confirm that those work?

**Comment by robertstrom** [comment]
I downloaded the arm64/aarch64 package and I have installed it. The program starts, is running the current version and appears to work. At this time I have not done any additional testing, but it does appear to at least work at the most basic level.

THANK YOU!

Will there be a link to these on this GitHub site along with the other releases, or will I have to bookmark this SUSE site?

On a slightly different note, whenever I perform an upgrade using a DEB package I always have to run a dkpg uninstall first - e.g.

sudo dpkg -r copyq-plugins
sudo dpkg -r copyq

and then
sudo dpkg -i copyq_7.1.0-1_arm64.deb

Not that it is a huge deal, but is there a way to just do an upgrade without having to do the remove first?

Thanks again!

Robert
